cmd/maya-apiserver/spc-watcher/storagepool_create.go

```diff
@@ -54,8 +55,24 @@ func CreateStoragePool(spcGot *apis.StoragePoolClaim, reSync bool, pendingPoolCo
 		return nil
 	}
 
+	// Get kubernetes clientset
```

Comment:
Maybe we should initialise the clientset in spcSync(), which is the caller function of createStoragePool and bind createStoragePool to the structure. As you suggested @sonasingh46 , it may also add value to rename the structure to a more apt name.

Reply:
Will rethink on this part... Need to discuss its packaging, code structure, naming conventions etc on a bigger picture with team. But will add a backlog for this.

Reply:
Ok